### ISSUE TITLE
Add default language for the web page

### ISSUE THREAD

**Issue by Dishebh** [fossasia/open-event-frontend#4243]
**Describe the bug**

If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by kushthedude** [comment]
I dont get the issue ? What does it want as a final result ?

**Comment by Dishebh** [comment]
> I dont get the issue ? What does it want as a final result ?

https://developer.paciellogroup.com/blog/2016/06/using-the-html-lang-attribute/

**Comment by kushthedude** [comment]
We already of localisation no sense of this.
